zerver/lib/test_helpers.py

```diff
@@ -373,7 +373,7 @@ def assert_json_error_contains(self, result, msg_substring, status_code=400):
         self.assertIn(msg_substring, self.get_json_error(result, status_code=status_code))
 
     def fixture_data(self, type, action, file_type='json'):
-        # type: (str, str, str) -> str
+        # type: (text_type, text_type, text_type) -> text_type
         return open(os.path.join(os.path.dirname(__file__),
                                  "../fixtures/%s/%s_%s.%s" % (type, type, action, file_type))).read()
```

Comment:
`open(...).read()` returns an `str`, even if the filename is `unicode`. We should use `force_text` here.